woocommerce/woocommerce
# Fix/29768

### All Submissions:






### Changes proposed in this Pull Request:



Closes #29768

### How to test the changes in this Pull Request:

1. Make sure to test this with PHP 8+.
2. Create a product and set an image.
3. Go to the customizer (Appearance->Customizer).
4. Navigate to WooCommerce->Product Images.
5. Set the cropping to custom and publish.
6. Go to your database and manually modify the option `woocommerce_thumbnail_cropping_custom_width` to a random string such as `abc` in the `wp_options` table.
7. Load this product up in the frontend. Ensure you do not get a fatal error.
8. Re-test everything this time with crop settings to `1:1`. Modify the option `woocommerce_thumbnail_cropping` to `1:abc` and also the other way around `abc:1`. Ensure you do not get a fatal error.

### Other information:




### Changelog entry

> Fix - PHP 8 error when cropping image values are not numeric.
